src/components/views/onboarding/styles.tsx

```diff
@@ -24,12 +23,11 @@ export const styles = StyleSheet.create({
   },
   groupOfPeople: {
     marginTop: -14,
-    // backgroundColor: colors.background,
     alignSelf: 'flex-start'
   },
   list: {
     flexDirection: 'row',
-    marginBottom: 16
+    marginBottom: 12
```

Comment:
Same: applied in multiple places? Is that correct? If this is fo the list that have a left border... is 12 the right values? Seems like it should be smaller? Designs looks like 2 (maybe 4?)

Reply:
same